book/website/collaboration/research-infrastructure-roles/community-manager.md

```diff
@@ -61,29 +80,31 @@ The CSCCE Skills Wheel include five core competancies: technical skills, communi
 See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
 ```
 
+<mark>Add more here from the TPR RCM paper on skills</mark>
+
 (cl-infrastructure-community-managers-challenges)=
-## Challenges for Community Managers 
+## Challenges for Research Community Managers 
 * Need to mediate between community members 
```

Comment:
Suggestion to replace the word 'need' throughout this section with 'required to'